src/tasks/processIssueComment/Repository/index.js

```diff
@@ -154,6 +156,24 @@ class Repository {
         await Promise.all(createOrUpdateFilesMultiple)
     }
 
+    async getPullRequestURL({ branchName }) {
+        try {
+            const results = await this.github.pulls.list({
+                owner: this.owner,
+                repo: this.repo,
+                state: 'open',
+                head: branchName,
+            })
+            return results.data[0].html_url
+        } catch (error) {
+            // Hard fail, but recoverable (not ideal for UX)
+            this.log.error(error)
+            throw new AllContributorBotError(
+                `A pull request is already open for the branch \`${branchName}\`.`,
+            )
+        }
```

Comment:
@jakebolam I know it's been a while, but do you by chance recall what this code was about? It's not tested, and the error message doesn't make sense to me